okta/help.md

```diff
@@ -27,6 +27,42 @@ The connection configuration accepts the following parameters:
 
 ### Actions
 
+#### Reset Password
+
+This action resets password for Okta user and transitions user status to PASSWORD_EXPIRED, so that the user is required to change their password at their next login
```

Comment:
```suggestion
This action resets password for Okta user and transitions user status to PASSWORD_EXPIRED, so that the user is required to change their password at their next login.
```